wildfly/connectors/infinispan/feature-pack-infinispan/src/main/resources/modules/system/layers/dv/org/jboss/teiid/resource-adapter/infinispan/hotrod/main/module.xml

```diff
@@ -19,5 +19,6 @@
         <module name="com.squareup"/> 				
         <module name="org.infinispan.client.hotrod" slot="teiid"/>
         <module name="org.infinispan.commons" slot="teiid"/>
+        <module name="org.infinispan.core" slot="teiid"/>
```

Comment:
I did not get the need for the "Infinispan-core" dependency. Is this for transaction stuff?

Reply:
Yes, it's where the wildfly transaction manager lookup is coming from.  Didn't see a great way to inject it otherwise.

Reply:
Cool, thanks for clarification.